csrc/mmdeploy/codebase/mmpose/CMakeLists.txt

```diff
@@ -2,11 +2,13 @@
 
 project(mmdeploy_mmpose)
 
+find_package(OpenCV REQUIRED)
```

Comment:
Do we have to find OpenCV?

Reply:
We used some functions from OpenCV in [keypoints_from_heatmap.cpp](https://github.com/open-mmlab/mmdeploy/blob/bede29dfb7cfc80713b13b9ad06b23ed9a49475a/csrc/mmdeploy/codebase/mmpose/keypoints_from_heatmap.cpp).